New capability/5.7/CIN-04031 (#55)
cinchy-co/Documentation: release-notes/release-notes/5.7-release-notes.md

We made various enhancements to the Connections Experience which should help to simplify and streamline your ability to create and maintain data synchronizations across the platform. Examples of these changes can be found in our [Data Sync documentation.](../../data-syncs/)

> #### Variable updates

Author: Here is the actual content for CIN-4031.

We've improved our variable system to boost clarity and reduce errors. Previously known as Parameters, the new design introduces three distinct variable types: Text, Formula, and Secret.

> Text Variables: This is the default type, functions like all variables before this update.

Reviewer: **suggestion:** "behaves the same way variables did before this update."

Author: ```suggestion
Text Variables: This is the default type, and behaves the same way variables did before this update.
```
